PSD to HTML first project/index.html

```diff
@@ -0,0 +1,195 @@
+            </p>
+            <a href="/" class="cta-button"
+              ><img src="img/phone-icon.png" class="phone-icon" />Download on
+              appstore</a
+            >
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section id="app-features">
+      <div class="container">
+        <h2 class="app-features">App Features</h2>
+        <p class="p-features">
+          Donut sesame snaps donut pie dessert chupa chups wafer dragée. Gummies
+          fruitcake chupa chups cookie liquorice. Chocolate pudding dessert
+          jelly fruitcake tart jelly wafer. Sweet roll carrot cake croissant.
+        </p>
+        <div class="row">
+          <div class="col-4-app wow fadeInLeft">
+            <img src="img/col-1-img.png" />
+            <h3>Scalable</h3>
+            <p>
+              Croissant ice cream macaroon cake. Dessert jelly beans candy
+              cookie. Bear claw candy canes gingerbread caramels cake. Macaroon
+              applicake gummi bears jelly marzipan.
+            </p>
+          </div>
+          <div class="col-4-app wow fadeInUp">
+            <img src="img/col-2-img.png" />
+            <h3>Instant</h3>
+            <p>
+              Croissant ice cream macaroon cake. Dessert jelly beans candy
+              cookie. Bear claw candy canes gingerbread caramels cake. Macaroon
+              applicake gummi bears jelly marzipan.
+            </p>
+          </div>
+          <div class="col-4-app wow fadeInRight">
+            <img src="img/col-3-img.png" />
+            <h3>Something</h3>
+            <p>
+              Croissant ice cream macaroon cake. Dessert jelly beans candy
+              cookie. Bear claw candy canes gingerbread caramels cake. Macaroon
+              applicake gummi bears jelly marzipan.
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section id="nose">
+      <div class="container">
+        <div class="download-info">
+          <h2>
+            <span class="reddish">Chocolate</span> pudding dessert jelly
+            <span class="reddish">fruit</span>
+            <span class="reddish">cake</span>
+            <span class="reddish">tart jelly wafer.
+          </h2>
+          <p class="nose-p">
+            Donut cupcake lemon drops dragée liquorice apple pie powder. Topping
+            sweet lollipop soufflé chocolate liquorice candy canes.
+          </p>
+          <a href="#" class="download-btn">Download</a>
+        </div>
+      </div>
+    </section>
+
+    <footer>
+      <div class="container">
+        <div class="logo-footer">
+          <img src="img/site-logo.png" class="footer-logo" />
+        </div>
+        <div class="sections">
+                    <div class="col-4-footer">
+                        <h4>Popular Posts</h4>
+                        <ul class="ul-posts">
+                            <li class="reddish-bg">
```

Comment:
1) img-to ne trqbva da e background a da si e `<img>`
2) vytre6no li-tata trqbva da sa linkove
3) ne izpolzvai takova stilvane za padane na nov red v takyv slu4ai. Izpolzvai `<br />`

Summary - html-a trqbva da izglejda taka:
```
<ul class="ul-posts">
    <li>
    	<img src="img/rectangle.png">
    	<h5>I love donut <br />bonbon cakes</h5>
    </li>
    <li>
    	<img src="img/rectangle.png">
    	<h5>Candy fruitcake biscuit chupa chups</h5>
    </li>
    <li>
    	<img src="img/rectangle.png">
    	<h5>Candy fruitcake biscuit chupa chups</h5>
    </li>
</ul>
```